src/components/Slider/Slider.tsx

```diff
@@ -0,0 +1,43 @@
+/* (c) Copyright Frontify Ltd., all rights reserved. */
+
+import { IconProps } from "@components/Icon/Icon";
+import { Dispatch, ReactElement, SetStateAction } from "react";
+import css from "./Slider.module.css";
+
+export interface IconItem {
+    id: string;
+    icon: ReactElement<IconProps>;
+}
+
+export interface TextItem {
+    id: string;
+    name: string;
+}
+
+export interface SliderProps {
+    items: TextItem[] | IconItem[];
+    activeItemId: string;
+    onChange: Dispatch<SetStateAction<string>>;
```

Comment:
```suggestion
    onChange: (id: string) => void;
```

We don't use Redux within the product